lib/project.ts

```diff
@@ -630,6 +630,18 @@ export class Project implements Project.IProject {
 		}).future<void>()();
 	}
 
+	public isTypeScriptProject(): IFuture<boolean> {
```

Comment:
I think we have similar function in `typescript-compilation-service`. Can we use it ?